en/components/getting_started.md

```diff
@@ -0,0 +1,299 @@
+
+### Create an application by using the Ignite UI CLI
+
+Let's start by opening a preferred terminal and installing the [`Ignite UI CLI`](https://github.com/IgniteUI/igniteui-cli):
+
+```cmd
+npm install -g igniteui-cli 
+```
+
+#### Using guided experience
+
+The shortest and easiest way to bootstrap an application is to use the Ignite UI CLI [`guided experience`](https://github.com/IgniteUI/igniteui-cli/wiki/step-by-step), which builds a configured app that the developer can get up and running with the ease of a single command. 
+
+```cmd
+ig
+```
+
+<div style="display:inline-block;">
+    <a style="background: url(../images/general/buildCLIapp.gif); display:flex; justify-content:center; min-width:540px; min-height:315px;"
+       href="https://youtu.be/QK_NsdtdA70" target="_blank">
+        <img src="../images/general/play.svg" style="vertical-align: middle;" />
+    </a>
+    <p style="text-align:center;">Building Your First Ignite UI CLI App</p>
+</div>
+
+#### Using commands
+
+We can always use the Ignite UI CLI [`commands`](https://github.com/IgniteUI/igniteui-cli/wiki#available-commands) for generating an Ignite UI project, adding a new component or building and serving the project by ourselves. For that purpose, we can use the following commands:
+
+```cmd
+ig new <project name> --framework=angular --type=igx-ts 
+cd <project name>
+ig add <component/template> <component_name>
+ig start 
+```
+
+- The full list of available **component/template** values is [`here`](https://github.com/IgniteUI/igniteui-cli/wiki/add#ignite-ui-for-angular-templates).
+- Both the **project name** and the **component_name** are custom values, provided by the developer.
+
+Here is an example of creating a new Ignite UI for Angular project with an igxGrid control:
+
+```cmd
+ig new newAngularProject --framework=angular --type=igx-ts
+cd newAngularProject
+ig add grid newGrid
+ig start
+```
+
+After executing those simple commands, your new project will be built and served. It will automatically open in your default browser and you will be able to inspect your grid control and to make changes to the project.
+
+### Create an application using the Angular CLI
+
+If you do not want to take advantage of the fully-automated process of Ignite UI for Angular projects creation in the Ignite UI CLI, and you prefer creating your project manually, using the [`Angular CLI`](https://github.com/angular/angular-cli) – you can achieve that relatively simply.
+
+#### Install Angular CLI
+
+First we will have to install the Angular CLI itself:
+
+```cmd
+npm install -g @angular/cli
+```
+
+#### Create project
+
+Now we can create our new project:
+
+```cmd
+ng new my-igniteui-project
+cd my-igniteui-project
+```
+
+#### Install Ignite UI for Angular
+
+To install Ignite UI for Angular to our project, we simply run:
+
+```cmd
+npm install igniteui-angular
```

Comment:
Replace this with `ng add igniteui-angular`.

Reply:
We need to remove them. Because since they can use ig add, they'll never choose to do the steps manually.

Reply:
Yeah, we want people to go through `ng add`, so they don't have to execute those exact steps. You can remove them.